src/amo/css/inc/vars.scss

```diff
@@ -18,6 +18,7 @@ $breakpoints: (
   medium: only screen and (min-width: 500px),
   large: only screen and (min-width: 720px),
   extraLarge: only screen and (min-width: 900px),
+  desktop: only screen and (min-width: 1025px),
```

Comment:
These breakpoints have been named after sizing, not device. Could we call it `extraWide` maybe?